src/Sylius/Bundle/UserBundle/Resources/config/services.xml

```diff
@@ -179,12 +182,12 @@
         <service id="sylius.form.type.user_change_password" class="%sylius.form.type.user_change_password.class%">
             <tag name="form.type" alias="sylius_user_change_password" />
         </service>
-        <service id="sylius.form.type.gender" class="Sylius\Bundle\UserBundle\Form\Type\GenderType">
+        <service id="sylius.form.type.gender" class="%sylius.form.type.gender.class%">
```

Comment:
I don't think this is correct, we should follow Symfony changes & not use class names as parameters.